src/cli/Cli.ts

```diff
@@ -988,10 +988,10 @@ export class Cli {
     return response;
   }
 
-  public static async handleMultipleResultsFound(promptMessage: string, errorMessage: string, values: { [key: string]: object }): Promise<object | CommandError> {
+  public static async handleMultipleResultsFound<T>(promptMessage: string, errorMessage: string, values: { [key: string]: T }): Promise<T> {
```

Comment:
Looking at this, it seems the `promptMessage` and `errorMessage` are now overlapping like 90%. Am I correct on that? 

I think it would be best if we just merge the two properties into one, and add the 'Please choose one' dynamically in the function.

What do you think? Do it now? Or do it in a separate refactor?

Reply:
Hmmm again a good point.
I think you might be right and in most cases it will be like this.
I would rather do it the right way already instead of leaving not perfect code in main branch 😉.
I will recheck and if possible refactor this today over the night. Thanks for the feed 👍.
Keep it coming 👏

Reply:
Did you get to this @Adam-it ?

Reply:
I am in the process of updating this. I already fixed up 50+ places where it is used but now I need to align the tests 🙂. Unfortunately the change is in many places so even a simple update takes time 😔. I hope to finish it up this week 👍

Reply:
Take your time 👍

Reply:
I rather not as this PR usually gets conflicted when anything else is merged 🙂

Reply:
✅